ansible/roles/docker_config/README.adoc

```diff
@@ -0,0 +1,9 @@
+== Role: Docker
+
+k8s_docker: Docker k8s configuration
```

Comment:
You should mention that the role will:
- Expose the TCP port 2376 of the docker daemon
- Add insecure-registries

Reply:
I cannot answer to what this role does as it already existed and I only renamed it.